presto-kafka/src/main/java/com/facebook/presto/kafka/KafkaTableDescriptionSupplier.java

```diff
@@ -129,11 +119,16 @@
         return ImmutableList.of();
     }
 
-    private static SchemaTableName parseTableName(String schemaTableName)
+    private SchemaTableName parseTableName(String schemaTableName)
     {
-        checkArgument(!isNullOrEmpty(schemaTableName), "schemaTableName is null or is empty");
-        List<String> parts = Splitter.on('.').splitToList(schemaTableName);
-        checkArgument(parts.size() == 2, "Invalid schemaTableName: %s", schemaTableName);
-        return new SchemaTableName(parts.get(0), parts.get(1));
+        try {
+            checkArgument(!isNullOrEmpty(schemaTableName), "schemaTableName is null or is empty");
+            List<String> parts = Splitter.on('.').splitToList(schemaTableName);
+            checkArgument(parts.size() == 2, "Invalid schemaTableName: %s", schemaTableName);
+            return new SchemaTableName(parts.get(0), parts.get(1));
+        }
+        catch (IllegalArgumentException iae) {
```

Comment:
Name the exception argument `ignored` so it's clear to the reader and IntelliJ that we are intentionally ignoring it.